### PR TITLE
Replace swsssdk.ConfigDBConnector and SonicDBConfig with swsscommon implementation in sonic-py-common

### PR DESCRIPTION
**Pull request by qiluo-msft**
#### Why I did it
swsssdk will be deprecated. Use swsscommon instead.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: src/sonic-py-common/sonic_py_common/multi_asic.py [hunk 1]
```diff
@@ -39,8 +37,8 @@ def connect_config_db_for_ns(namespace=DEFAULT_NAMESPACE):
     Returns:
       handle to the config_db for a namespace
     """
-    SonicDBConfig.load_sonic_global_db_config()
-    config_db = ConfigDBConnector(namespace=namespace)
+    swsscommon.SonicDBConfig.load_sonic_global_db_config()
+    config_db = swsscommon.ConfigDBConnector(namespace=namespace)
```
Comment (qiluo-msft) [inline src/sonic-py-common/sonic_py_common/multi_asic.py:41]: >swsscommon [](http://example.com/codeflow?start=16&length=10)

This change will lead to some error message
```
admin@sonic:~$ sudo config load_minigraph
Warning: failed to retrieve PORT table from ConfigDB!
Traceback (most recent call last):
  File "/usr/local/bin/config", line 8, in <module>
    sys.exit(config())
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 1134, in invoke
    Command.invoke(self, ctx)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/config/main.py", line 913, in config
    SonicDBConfig.initialize()
RuntimeError: SonicDBConfig already initialized
```

**Comment by liuh-80** [comment]
azp run

**Comment by liuh-80** [comment]
This PR closed, I will continue this work with another PR: https://github.com/Azure/sonic-buildimage/pull/10099
Because: 1. merge conflict in this PR. 2. the PR is based on a branch in Qi's repo.

**Pull request closed without merging**